.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
   - sudo apt-get -qq install autoconf automake bison e2fslibs-dev libfuse-dev libtool liblzma-dev gettext
 
 install:
-  - make install.tools
+  - make install.libseccomp.sudo install.tools
```

Comment:
The Travis tests are all containerized, so these both do nothing

Reply:
Need to add install.libseccomp.sudo to the Dockerfile and remove `libseccomp` from the things apt installs in the dockerfile